src/main.rs

```diff
@@ -25,6 +26,10 @@ use structopt::StructOpt;
 enum Cli {
     /// Get wallet information
     Info {
+        /// Whether to use Ledger
```

Comment:
Since this is also the commad line help text, something like "Use Ledger hardware wallet` might sound better.